msticnb/nb/azsent/network/network_flow_summary.py

```diff
@@ -118,6 +118,8 @@ class NetworkFlowSummary(Notebooklet):
 
     # assign metadata from YAML to class variable
     metadata = _CLS_METADATA
+    __doc__ = nb_metadata.update_class_doc(__doc__, metadata)
+    _cell_docs = _CELL_DOCS
```

Comment:
I'm a bit confused about what this is doing given that _cell_docs doesn't seem to be referenced anywhere else.

Reply:
Yeah it is a bit obscure - what it's doing is taking some of the data from the metadata yaml and adding it to the class doc string. I think this only works at runtime so not sure if Sphinx will pick it up (although it's in the class top-level code, so should be run when the class is imported, which is what Sphinx does). It passes the existing doc string to the function and appends options docs to the thing - so we don't have to document them in multiple places.